selfdrive/controls/controlsd.py

```diff
@@ -121,6 +121,11 @@ def __init__(self, CI=None):
       safety_config.safetyModel = car.CarParams.SafetyModel.noOutput
       self.CP.safetyConfigs = [safety_config]
 
+    # Write previous route's CarParams
+    prev_cp = self.params.get("CarParamsPersistent")
+    if prev_cp is not None:
```

Comment:
don't you also want to check if it's the same car?

Reply:
The checks are handled in the daemons (there may be different checks). I just need access to the previous carParam from one source.